### DISCUSSION TITLE
Japanese translation problem: highlighted lines mismatch with English ver.

### DISCUSSION THREAD

**Discussion by ftnext** [fastapi/fastapi#6730] [Questions]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [ ] I searched the FastAPI documentation, with the integrated search.
* [ ] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [ ] I already read and followed all the tutorial in the docs and didn't find an answer.
* [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

Thanks a lot to Japanese translation.
I found bugged highlighted lines in https://fastapi.tiangolo.com/ja/tutorial/query-params/ .

### Example

Highlighted empty line

  - https://fastapi.tiangolo.com/ja/tutorial/query-params/#_3
  - https://fastapi.tiangolo.com/ja/tutorial/query-params/#_4
  - https://fastapi.tiangolo.com/ja/tutorial/query-params/#_5
  - https://fastapi.tiangolo.com/ja/tutorial/query-params/#_6 (Second Python code)

### Description

It seems to me that the translated file needs to be synced with English version.

- translated: fastapi/docs/en/docs/tutorial/query-params.md
- English version: fastapi/docs/ja/docs/tutorial/query-params.md

About docs_src/query_params/tutorial002.py

- As is: `hl_lines="7"` (ref: fastapi/docs/en/docs/tutorial/query-params.md line 67)
- To be: `hl_lines="9"` (ref: fastapi/docs/ja/docs/tutorial/query-params.md line 66)

About docs_src/query_params/tutorial003.py

- As is: `hl_lines="7"`
- To be: `hl_lines="9"`

About docs_src/query_params/tutorial004.py

- As is: `hl_lines="6  8"`
- To be: `hl_lines=hl_lines="8  10"`

### Additional context

I checked only "Query parameters" in Japanese translation.
Sorry, I don't know if we need to fix the highlighted lines for other pages in the Japanese translation or for translations in other languages right now.

**Comment by tokusumi** [comment]
@ftnext 
Some translated documents is behind the latest English documents because update on translation takes more time and needs more contributors than main documents. 
Especially now, there are few Japanese contributor check outdated lines.
So, really welcome for your pull requests about other pages as well as this page.
